Gemfile

```diff
@@ -3,11 +3,3 @@ source 'https://rubygems.org'
 gemspec
 
 gem 'ruby-fifo', github: 'shurizzle/ruby-fifo'
-
-group :development do
-  gem 'rspec'
```

Comment:
> BTW, I found CI failed once or twice due to a timing issue. 

I know the problem, the code which will wait for an answer when a new time base is set, should be added. But it is as far as I can